package.json

```
@@ -240,73 +240,74 @@
-		"precheck-local-changes": "npm run docs:build",
-		"check-local-changes": "node ./bin/check-local-changes.js",
+		"clean:package-types": "tsc --build --clean",
+		"clean:packages": "rimraf \"./packages/*/@(build|build-module|build-style)\"",
 		"dev": "npm run build:packages && concurrently \"wp-scripts start\" \"npm run dev:packages\"",
 		"dev:packages": "concurrently \"node ./bin/packages/watch.js\" \"tsc --build --watch\"",
 		"distclean": "rimraf node_modules packages/*/node_modules",
+		"docs:blocks": "node ./bin/api-docs/gen-block-lib-list.js",
+		"docs:build": "npm-run-all docs:gen docs:blocks docs:ref docs:theme-ref",
 		"docs:gen": "node ./docs/tool/index.js",
-		"docs:build": "npm-run-all docs:gen api-docs:*",
+		"docs:ref": "node ./bin/api-docs/update-api-docs.js",
+		"docs:theme-ref": "node ./bin/api-docs/gen-theme-reference.js",
+		"env": "wp-env",
 		"fixtures:clean": "rimraf \"test/integration/fixtures/blocks/*.+(json|serialized.html)\"",
-		"fixtures:generate": "cross-env GENERATE_MISSING_FIXTURES=y npm run test-unit test/integration/full-content/ && npm run format test/integration/fixtures/blocks/*.json",
-		"fixtures:regenerate": "npm run fixtures:clean && npm run fixtures:generate",
+		"fixtures:generate": "cross-env GENERATE_MISSING_FIXTURES=y npm run test:unit test/integration/full-content/ && npm run format test/integration/fixtures/blocks/*.json",
+		"fixtures:regenerate": "npm-run-all fixtures:clean fixtures:generate",
 		"format": "wp-scripts format",
-		"format-php": "wp-env run composer run-script format",
-		"lint": "concurrently \"npm run lint-lockfile\" \"npm run lint-js\" \"npm run lint-pkg-json\" \"npm run lint-css\"",
-		"lint-js": "wp-scripts lint-js",
-		"lint-js:fix": "npm run lint-js -- --fix",
-		"prelint-php": "wp-env run composer \"install --no-interaction\"",
-		"lint-php": "wp-env run composer run-script lint",
-		"lint-pkg-json": "wp-scripts lint-pkg-json . 'packages/*/package.json'",
-		"lint-lockfile": "node ./bin/validate-package-lock.js",
-		"lint-css": "wp-scripts lint-style \"**/*.scss\"",
-		"lint-css:fix": "npm run lint-css -- --fix",
-		"lint:md-docs": "wp-scripts lint-md-docs",
+		"format:php": "wp-env run composer run-script format",
+		"lint": "concurrently \"npm run lint:lockfile\" \"npm run lint:js\" \"npm run lint:pkg-json\" \"npm run lint:css\"",
+		"lint:css": "wp-scripts lint-style \"**/*.scss\"",
+		"lint:css:fix": "npm run lint:css -- --fix",
+		"lint:js": "wp-scripts lint-js",
+		"lint:js:fix": "npm run lint:js -- --fix",
+		"lint:lockfile": "node ./bin/validate-package-lock.js",
+		"lint:md:docs": "wp-scripts lint-md-docs",
+		"prelint:php": "wp-env run composer \"install --no-interaction\"",
+		"lint:php": "wp-env run composer run-script lint",
+		"lint:pkg-json": "wp-scripts lint-pkg-json . 'packages/*/package.json'",
 		"native": "npm run --prefix packages/react-native-editor",
+		"other:analyze-bundles": "npm run build -- --webpack-bundle-analyzer",
```

This is related to build, so it should have the same group:
```suggestion
		"build:analyze-bundles": "npm run build -- --webpack-bundle-analyzer",
```

---

Fixed with 44c9ba9